MozillaFoundation/foundation.mozilla.org
# Remove fonts from sass build workflow

# Description

This PR changes the static font includes from the .scss workflow to static.css

Since Sass isn't necessary for something so infrequently updated as fonts, it should result in build time similar to before we were using static fonts. 

Link to sample test page: https://foundation-s-tp1-685-st-owmknf.herokuapp.com/
Related PRs/issues: #12362

# To Test
1) Checkout main via `git checkout main`
2) Run `inv npm "run build"`
3) Note that the fonts are part of the postcss Sass procesing (i.e. `fonts.compiled.css`)
4) Checkout `tp1-685-static-css-fonts` via `git checkout tp1-685-static-css-fonts`
5) Run `inv npm "run build"`
6) Note that the fonts are no longer part of the postcss Sass procesing (i.e. `fonts.compiled.css`)
7) View the [review app](https://foundation-s-tp1-685-st-owmknf.herokuapp.com/) and note there is no font regression. 8) Inspect the page, view network resources, and note the new css file is loading `base.min.(somerandomhash).css`


┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-753)
